tests/nnfw_api/lib/GenModelTrain.h

```diff
@@ -84,10 +84,9 @@ struct TrainCaseData
    * @tparam T Data type
    * @param data vector data array
    */
-  TrainCaseData &setLosses(std::vector<float> losses)
+  TrainCaseData &addLosses(const std::vector<float> &loss)
   {
-    this->losses = losses;
-
+    addLoss(losses, loss);
```

Comment:
Is there a reason to define `addLoss` as a separate function? 

```suggestion
    this->losses.emplace_back(loss);
```

Reply:
There is no particular reason. I just wanted to keep the same level of function call. Your suggestion is reasonable.

Reply:
> I just wanted to keep the same level of function call. 

Aha, I also understood your intention :smile: